cltk/tests/test_tokenize.py

```diff
@@ -91,6 +91,7 @@ def test_latin_word_tokenizer(self):
         # - Lucr. DRN. 5.1351-53
         # - Plaut. Bacch. 837-38
         # - Plaut. Amph. 823
```

Comment:
Better to place each comment after each quote with a documentation comment:
#: Lucretius, De Rerum Natura 5.1351-1353

Reply:
Hi thank you for your review. I only added one testline to this suit and try to adapt to the style of the others. Should I still change them? ;)